packages/web-components/fast-components/src/horizontal-scroll/fixtures/horizontal-scroll.html

```diff
@@ -0,0 +1,301 @@
+<h1>Horizontal scroll</h1>
+
+<style type="text/css">
+    fast-card {
+        width: 120px;
+        height: 200px;
+        margin: 5px;
+    }
+
+    fast-horizontal-scroll {
+        max-width: 620px;
+        margin: 20px;
+    }
+
+    .right-gradient {
+        --scroll-fade-next: var(--background-color);
+    }
+
+    .both-gradient {
+        --scroll-fade-next: var(--background-color);
+        --scroll-fade-previous: var(--background-color);
```

Comment:
Are the terms "previous" and "next" too objective, considering in RTL that may mean the cards advance to the left?

Reply:
I used the names next and previous to account for RTL. The next flipper will scroll left in LTR and scroll right in RTL. I was also matching the naming used in the fast-flipper control for consistency.